pyproject.toml

```diff
@@ -76,11 +77,17 @@ rocm624 = [
     "pytorch-triton-rocm>=2.0.0",
 ]
 
-deepsort = [
-    "scipy>=1.13.1",
+reid = [
+    "safetensors>=0.5.3",
     "timm>=1.0.15",
 ]
```

Comment:
`deepsort/tracker.md` still contains instructions referencing the `deepsort` extra dependencies. It looks like we need to migrate those to `reid`.